cadf/dispatcher.sls

```diff
@@ -42,14 +52,5 @@ cron_path:
     - require:
       - pkg: cadf_packages
 
-{%- else %}
-
-distpatcher_cron:
-  cron.absent:
-    - identifier: cadf_dispatcher
-    - user: "{{ config.get('user','root') }}"
-    - require:
```

Comment:
I would only remove the require here.
As cront.absent should not fire anny errors if the task is no longer in list?

Reply:
It's fine with us. As after removed condition for cadf_package (require option), it doent throw error for other nodes where dispatcher disabled.